pulsar-client-api/src/main/java/org/apache/pulsar/common/utils/IOUtils.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utils for I/O operations.
+ */
+public final class IOUtils {
```

Comment:
This implement is copied from JDK.

Reply:
You don't need to change, this is a prompt message.

Reply:
Do you mean it was copied from JDK 9+ `InputStream.readAllBytes()`?

Would it be useful to put a comment in the code the describe it as such? Perhaps also a TODO to remove it when the client code compilation target is upgraded to >= 9.

Reply:
commented